ci/local/build.sh

```diff
@@ -90,6 +91,10 @@ fi
 mkdir -p ${BASE_CONTAINER_BUILD_DIR}
 mkdir -p ${CPP_CONTAINER_BUILD_DIR}
 mkdir -p ${PYTHON_CONTAINER_BUILD_DIR}
+# Create build directories. This is to ensure correct owner for directories. If
+# directories don't exits docker volume mounting side effect is that build directorier would be owned by root
+mkdir -p "${REPO_PATH}/build"
```

Comment:
you should use `${CPP_BUILD_DIR}` and `${PYTHON_BUILD_DIR}` for consistency.

Reply:
Fixed on latest iteration. Uses
mkdir -p "${REPO_PATH}/${CPP_BUILD_DIR}"
mkdir -p "${REPO_PATH}/${PYTHON_BUILD_DIR}"